libraries/src/Cache/CacheController.php

```diff
@@ -18,6 +18,7 @@
  *
  * @since  1.7.0
  * @note   As of 4.0 this class will be abstract
+ * @mixin Cache Proxy all CacheController absent method calls to Cache
```

Comment:
cs
```suggestion
 * @mixin  Cache Proxy all CacheController absent method calls to Cache
```

But if I'm not wrong you have to list all classes that are mixin?

Reply:
No. CacheController proxifies only Cache methods via __call. This comment helps with IDE.